vms/platformvm/utxo/camino_handler_test.go

```diff
@@ -106,3 +109,421 @@ func TestVerifySpendUTXOsWithLocked(t *testing.T) {
 		})
 	}
 }
+
+func TestCaminoVerifySpendUTXOs(t *testing.T) {
+	fx := &secp256k1fx.Fx{}
+
+	err := fx.InitializeVM(&secp256k1fx.TestVM{})
+	require.NoError(t, err)
+
+	err = fx.Bootstrapped()
+	require.NoError(t, err)
+
+	h := &handler{
+		ctx: snow.DefaultContextTest(),
+		clk: &mockable.Clock{},
+		utxosReader: avax.NewUTXOState(
+			memdb.New(),
+			txs.Codec,
+		),
+		fx: fx,
+	}
+
+	tx := &dummyUnsignedTx{txs.BaseTx{}}
+	tx.Initialize([]byte{0})
+
+	credentials := generateSignersWithKeys(tx, keys)
+
+	address0 := keys[0].PublicKey().Address()
+	address1 := keys[1].PublicKey().Address()
+	address2 := keys[2].PublicKey().Address()
+
+	tests := map[string]struct {
+		utxos           []*avax.UTXO
+		ins             []*avax.TransferableInput
+		outs            []*avax.TransferableOutput
+		creds           []verify.Verifiable
+		signers         []verify.State
+		producedAmounts map[ids.ID]uint64
+		expectedErr     error
+	}{
+		"No Inputs, No Outputs": {
```

Comment:
Remove - this is a task for ava / general case test (if necessary should be in `handler_test`)